viewing_party/main.py

```diff
@@ -0,0 +1,157 @@
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title and genre and rating:
+        new_movie["title"] = movie_title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
```

Comment:
this works just fine! but one way to shorten this into one line:

```suggestion
    if movie_title and genre and rating:
        new_movie = {"title": movie_title, "genre": genre, "rating": rating}
```